subtitles/zh-CN/10_instantiate-a-transformers-model-(pytorch).srt

```diff
@@ -10,107 +10,107 @@
 
 3
 00:00:08,483 --> 00:00:11,790
-在本视频中，我们将了解如何创建用户模型
+在本视频中，我们会带您了解如何
 In this video, we'll look at how we can create a user model
 
 4
 00:00:11,790 --> 00:00:13,290
-来自变形金刚图书馆。
+用 Transformers 库创建用户模型。
 from the Transformers library.
 
 5
 00:00:14,310 --> 00:00:17,100
-正如我们之前看到的 AutoModel 类允许
+正如我们之前看到的，AutoModel 类允许
 As we have seen before the AutoModel class allows
 
 6
 00:00:17,100 --> 00:00:19,140
-你实例化一个预训练模型
+您从 Hugging Face Hub 的任何检查点
 you to instantiate a pretrained model
 
 7
 00:00:19,140 --> 00:00:21,513
-从 Hugging Face Hub 上的任何检查站。
+去实例化一个预训练的模型。
 from any checkpoint on the Hugging Face Hub.
 
 8
 00:00:22,350 --> 00:00:23,910
-它会选择正确的模型类
+AutoModel 类会从开源库中挑选合适的模型类
 It'll pick the right model class
 
 9
 00:00:23,910 --> 00:00:26,654
-从库中实例化适当的体系结构
+去实例化对应的架构
 from the library to instantiate the proper architecture
 
 10
 00:00:26,654 --> 00:00:29,793
-和大量的权重作为内部的预训练模型。
+和权重负载来作为预训练模型。
 and loads of weights as the pretrained model inside.
 
 11
 00:00:30,690 --> 00:00:33,810
-正如我们所见，当给定一个 BERT 检查点时
+正如我们所见，当给定一个 BERT 检查点时，
 As we can see, when given a BERT checkpoint
 
 12
 00:00:33,810 --> 00:00:38,043
-我们最终得到一个 BertModel，类似地，对于 GPT-2 或 BART。
+我们最终会得到一个 BertModel ，类似地，模型GPT-2 或 BART 也可以这么做。
 we end up with a BertModel and similarly, for GPT-2 or BART.

```

Comment:
```suggestion
we end up with a BertModel and similarly, for GPT-2 or BERT.

```

This is an issue of the original English subtitle, could you please open a PR for this?